src/bun.js/bindings/JSNextTickQueue.cpp

```diff
@@ -84,7 +84,7 @@ void JSNextTickQueue::drain(JSC::VM& vm, JSC::JSGlobalObject* globalObject)
 
     if (!isEmpty()) {
         if (mustResetContext) {
-            globalObject->m_asyncContextData.get()->putInternalField(vm, 0, jsUndefined());
+            globalObject->asyncContextTuple()->putInternalField(vm, 0, jsUndefined());
         }
```

Comment:
With this PR I've made a distinction between the asyncContextTuple and asyncContextData because in main the current forms are conflated as `asyncContext` which gets confusing (is it the tuple or the data of the tuple). With this we have a clear and consistent distinction.